romsel_aktheme/arm9/source/language.h

```diff
@@ -53,7 +53,9 @@ class LanguageFile : public CIniFile
     case DSiMenuPlusPlusSettings::ELangItalian:
       return "lang_it";
     case DSiMenuPlusPlusSettings::ELangChinese:
-      return "lang_cn";
+      return "lang_cn";
+	case DSiMenuPlusPlusSettings::ELangKorean:

```

Comment:
Indent?

Reply:
> Indent?

Ha, it is fine on my PC.
idk why it is indented after upload to github.
And I will fix it.